libs/wingsdk/src/std/index.ts

```diff
@@ -9,3 +9,4 @@ export * from "./resource";
 export * from "./set";
 export * from "./string";
 export * from "./util";
```

Comment:
Do we really need this export?

Reply:
I assumed it's required for the compiler, we're moving it to a different namespace so I think we should be alright